ublox/tests/parser_tests.rs

```diff
@@ -322,3 +322,22 @@ fn test_double_start_at_end() {
     }
     assert!(it.next().is_none());
 }
+
+#[test]
+fn test_ack_ack_to_owned_can_be_moved() {
+    let ack_ack = [0xb5, 0x62, 0x5, 0x1, 0x2, 0x0, 0x4, 0x5, 0x11, 0x38];
+
+    let mut parser = ublox::Parser::default();
+    let mut it = parser.consume(&ack_ack);
+    match it.next() {
+        Some(Ok(PacketRef::AckAck(ack_packet))) => {
+            assert_eq!(ack_packet.class(), 4); // Why is this 4 ???
```

Comment:
@CramBL, could you remove the comment. Or better, put a comment on top saying that `4` is the class id of the acknowledged packet from the payload of UbxAckAck, as a reminder for ourselves.

Reply:
Yes of course. Is there any other tests that should be implemented in your view?

Reply:
I don't really have something else in mind for this currently. :(